amd/src/record_handler.js

```diff
@@ -0,0 +1,199 @@
+define(['jquery'], function ($) {
+    return {
+        init: function (room, instrutor, language) {
+
```

Comment:
@leooAprigioo please make sure that all URLs/IPs are variables.

Reply:
- url_jitsi -> domain jitsi option
- url_recording_service -> (new) recording domain option

Reply:
NOTE: For jitsi I think it's okay use only one domain because of HAProxy, need to check with @guizanelato for the recording service (not sure how the API will handle multi machine orchestration).